Sources/Target/POSIX/ELFProcess.cpp

```diff
@@ -203,10 +201,11 @@ EnumerateLinkMap(ELFProcess *process, Address addressToDPtr,
     if (error != ds2::kSuccess)
       return error;
 
+    shlib.main = (process->loadBase() == linkMap.baseAddress);
```

Comment:
Discussed this offline with Francis: this will work for PIE executables, and I think that's better than the current solution so I'm fine accepting this diff. And doing the right thing here probably requires changing lldb.